Updated Footer component and made some minor changes to Landing and Landing Navbar.js

Solemai opened the pull request:
Updated footer component to be added to the other pages. More functionality could be added to it. 
Also made some minor CSS changes to Landing and Landing Navbar.js.

Solemai committed:
Navbar Move

Moved JS and CSS Navbar code from Landing.JS to LandingNavBar.js to separate.

Solemai committed:
Updated Social Icon Images

Updated the image links in the navigation footer.
Rename images and class name for clarity in JS and CSS code

Solemai committed:
Fixing Images under subscription feature section

Updated naming in JS and CSS code. Fixing the formatting for subscription feature description 3 and 4

Solemai committed:
Merge branch 'main' into LandingPage3.0-LR

Solemai committed:
Merge branch 'main' into LandingPage3.0-LR

Solemai committed:
Footer and Landing Navbar Edits

Moved footer code from Landing.js to Footer,js component.

Updated colors on navbar component.

Solemai committed:
Merge branch 'main' into LandingPage3.0-LR

Solemai committed:
Merge branch 'main' into LandingPage3.0-LR

Solemai committed:
Updated footer and minor changes on landing navbar

Option to add more funcitionality to footer component.

Made some minor changes to CSS for landing/landing navbar.

locationcyberia reviewed (changes requested):
Everything functions as supposed to -- only changes needed are cosmetic/styling.

- Navbar needs to be updated to ensure proper ratios.
- Footer needs to be updated to ensure permanent placement at the bottom of the screen.

abopoku commented:
The footer looks great! Cant wait to see it on each page once merged.
As far as the Navbar remember it has been permanently added to each page after I merged all the different ones. I have recently made updates to it as well so you can remove the personal navbar on the landing page.

Solemai committed:
Changes to Navbar, Footer, Landing, and CSS styling to all pages

Updated DangoDot header logo to link to landing page on Navbar.
Removed link to Landing.js from Navbar.
Added comments to the Global CSS file.
Updated headings and subscription features copy of landing pages.
Moved Navbar and Footer to App.js and removed it from all pages.
Updated styling on Landing, Home, Navbar, and Login for readability and to fit footer on pages.

Update signup today link on Landing.js to the Login.js

Solemai committed:
Added image carousel component for landing page

Downloaded Bootstrap and installed any necessary packages for image carousel.
Added four images for image carousel.
Added import React to Footer component.
Added image carousel component to landing page.
